integration/kubernetes/k8s-job.bats

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+load "${BATS_TEST_DIRNAME}/../../lib/common.bash"
+
+setup() {
+	export KUBECONFIG="$HOME/.kube/config"
+	get_pod_config_dir
+}
+
+@test "Run a job to completion" {
+	job_name="job-pi-test"
+	wait_time=60
+	sleep_time=2
+
+	# Create job
+	kubectl apply -f "${pod_config_dir}/job.yaml"
+
+	# Verify job
+	kubectl describe jobs/"$job_name" | grep "SuccessfulCreate"
+
+	# List pods that belong to the job
+	pod_name=$(kubectl get pods --output=jsonpath='{.items[*].metadata.name}')
+
+	# Verify that the job is completed
+	cmd="kubectl get pods -o jsonpath='{.items[*].status.phase}' | grep Succeeded"
+	waitForProcess "$wait_time" "$sleep_time" "$cmd"
+
+	# Verify the output of the pod
+	pi_number="3.14"
+	kubectl logs "$pod_name" | grep "$pi_number"
+}
+
+teardown() {
+	kubectl delete pod "$pod_name"
+	cmd="kubectl delete jobs/"$job_name" | grep deleted"
+	waitForProcess "$wait_time" "$sleep_time" "$cmd"
```

Comment:
this is going to call the `delete` action every 2 seconds... do we want that or we only want to call `delete` once and then check that the job no longer exists?